src/app/frontend/error/internalerror_controller.js

```diff
@@ -66,16 +70,13 @@ export class InternalErrorController {
    * @return {string} URL of GitHub page used to report bugs.
    */
   getLinkToFeedbackPage() {
-    let title = `Dashboard reported ${this.getErrorStatus()}`;
-    let body = `#### Issue details\n\n##### Environment\n<!-- Describe how do you run Kubernetes ` +
-        `and Dashboard.\n      Versions of Node.js, Go etc. are needed only from developers. To ` +
-        `get them use console:\n      $ node --version\n      $ go version\n-->\n\n\`\`\`\n` +
-        `Dashboard version:\nKubernetes version:\nOperating system:\nNode.js version:\nGo version` +
-        `:\n\`\`\`\n##### Steps to reproduce\n<!-- Describe all steps needed to reproduce the ` +
-        `issue. It is a good place to use numbered list. -->\n\n##### Observed result\nDashboard ` +
-        `reported ${this.getErrorStatus()}:\n\`\`\`\n${this.getErrorData()}\n\`\`\`\n\n##### ` +
-        `Expected result\n<!-- Describe expected result as precisely as possible. -->\n\n##### ` +
-        `Comments\n<!-- If you have any comments or more details, put them here. -->`;
+    let title = ``;
+    let body = `##### Steps to reproduce\n<!-- Describe all steps needed to reproduce the ` +
```

Comment:
It doesn't match Dashboard's issue template, but it may be better, because most of the users do not use Dashboard directly from their workspace but in container.

You may also add missing headers and spaces while using `<!-- Comment here. -->` comments to make it prettier.